src/components/PopupWithVerify.js

```diff
@@ -0,0 +1,23 @@
+import Popup from "./Popup.js";
+
+export default class PopupWithVerify extends Popup {
+  constructor(popupSelector) {
+    super(popupSelector);
+  }
+
+  setHandleDeleteMethod(handleDeleteSubmit) {
```

Comment:
Great job implementing confirmation popup and the functionality related to it! In general, it can support any confirmation.